core/src/test/java/com/sequenceiq/cloudbreak/controller/validation/network/MultiAzValidatorTest.java

```diff
@@ -80,6 +83,17 @@ public void testAwsWhenMultipleSubnetUsedShouldCauseValidationError() {
         Mockito.verify(builder, Mockito.times(1)).error(anyString());
     }
 
+    @Test
+    public void testSupportedForInstanceMetadataGenerationWhenPlatformSupportedShouldReturnTrue() {
```

Comment:
Perhaps add one more testcase for the negative result (or make this a parameterized test).